src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -89,37 +84,27 @@
 
 @Component
 @Profile("google")
-public class BigQueryPdao implements PrimaryDataAccess {
```

Comment:
Sorry, I'm not quite following why we want to get rid of the interface here

Reply:
The reason is that we won't be able to have an abstract interface to the tabular data store across different clouds.
We had that idea originally when our model was separate deployments for separate clouds. As we got experience with how the flight steps break down, it became clear that the interface wouldn't work. In the current code, probably half the calls to the Pdao do not use the interface. See [DR-262](https://broadworkbench.atlassian.net/browse/DR-262).